bootstrap.php

```diff
@@ -459,6 +459,31 @@ function ($app) {
     }
 );
 
+$app['place_history_projector'] = $app->share(
+    function ($app) {
+        $projector = new \CultuurNet\UDB3\Place\ReadModel\History\HistoryProjector(
+            $app['places_history_repository']
+        );
+        return $projector;
+    }
+);
+
+$app['places_history_repository'] = $app->share(
+    function ($app) {
+        return new \CultuurNet\UDB3\Doctrine\ReadModel\CacheDocumentRepository(
+            $app['place_history_cache']
```

Comment:
Nit: I'm not sure we need to create a separate service in the service container for this.
Could just be `$app['cache']('place_history');`